tests/plugins/test_koji.py

```diff
@@ -44,6 +44,10 @@ class X(object):
     pass
 
 
+class Y(object):
+    def __init__(self):
+        self.base_from_scratch = False
```

Comment:
Maybe `StubInsideBuilder` could be expanded to provide this attribute, and a `.with_base_from_scratch()` method to initialise it.

Reply:
What about the changes in this PR where we are introducing use of an InsideBuilder stub named 'Y'? Couldn't they use StubInsideBuilder instead?

Reply:
ok